Clarify use of alarm rvs values

emiltin opened the pull request:
Clarified that rvs is only filled in for active alarms, and all rvs values must be send, not only changed values.

Addresses #218.

However, it seems the documentation is inconsistent regarding whether the 'rvs' attribute should be included.

Can we require the rvs attribute to be present for active alarms, and left our for all other alarm types (inactive, request, acknowledge, suspend/resume, etc)?

emiltin committed:
Clarify use of alarm rvs values

emiltin commented on source/applicability/basic_structure.rst:
```diff
@@ -155,6 +156,13 @@ Alarm messages are event driven and sent to the supervision system
 messages are interaction driven.
 
+Details about an active alarm is provided in the `rvs` (Return ValueS) array.
+For all other alarm types, the `rvs` array must be empty. This includes when
```
can we leave out the rvs attribute instead of sending an empty array? the docs already leaves if out for some types, although it seems inconsistent or at least not well described

otterdahl replied:
I think the most important thing is that we are consistent. I suppose we can leave it out, after all it is a bit more cleaner. But I wonder if it may cause any unnecessary issues with backwards compatibility. In other words, is it worth it?

emiltin replied:
For minor 3.3.0 update we cannot remove it.

otterdahl commented on source/applicability/basic_structure.rst:
```diff
@@ -133,6 +133,7 @@ An alarm message is sent to the supervision system when:
 - An alarm is requested
 - An alarm is acknowledged
 - An alarm is being suspended / un-suspended
+- An active alarm is updated (ie. the return values change)
```
Have we written somewhere that alarms can not be repeated? E.g. do we need to update somewhere else too?

emiltin commented:
Updated the PR so it's based on 3.3.0. Also changed the text we're not adding recommendation rather than new requirements, to adhere to semver for a minor version:

- recommends complete rvs snapshots for active alarms.
 -recommends [] for inactive alarms, but requires receivers to accept conforming non-empty arrays.
- makes rvs-only updates optional.
- recommends identifying alarms using cId + aCId, not rvs.
